### DISCUSSION TITLE
Jamf Pro Deployment Guide

### DISCUSSION THREAD

**Discussion by MacAdmin5057** [IBM/mac-ibm-notifications#60] [Q&A] [answered]
Is it possible that the admins here can provide a Step by step admin guide to deploy in Jamf Pro. 

A great example (Mac Admins page to nudge (https://github.com/macadmins/nudge/wiki/Jamf-Pro-Guide)

I would be very thankful for this :)

**Comment by SMartorelli** [comment]
Hi @MacAdmin5057, thanks for the feedback! May I ask you which are the challenges you're facing deploying IBM Notifier to Jamf Pro?
In the meantime I leave [here](https://www.youtube.com/watch?v=Cn5wIuB90t8&list=PLlxHm_Px-Ie1EIRlDHG2lW5H7c2UYvops&index=14) the link to one of the JNUC 2021 sessions where our team did a little showcase of how we integrate IBM Notifier in Jamf Pro.

**Comment by mm2270** [comment, marked as answer]
Deploying IBM Notifier through Jamf Pro is pretty easy. I just created a package in Composer of where I wanted the app to go (in our case, a custom path within /private/var/ and then upload that pkg into Jamf and created a policy to push it out.

As an add-on, I have an Extension Attribute that captures the version of IBM Notifier installed, so that when new versions come out and I verify they are good to go, I can use a Smart Computer Group to target machines not on the new version I'm deploying.
